style.css

```diff
@@ -0,0 +1,68 @@
+.header-section {
+  display: flex;
+  justify-content: space-between;
+  align-items: baseline;
+}
+
+body {
+  position: absolute;
+  width: 375px;
+  height: 718px;
+  left: 0;
+  top: 1px;
+  background: #fff;
+}
+
+h1 {
+  width: 327px;
+  height: 150px;
+  font-family: Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.text1 {
+  width: 327px;
+  height: 146px;
+  font-family: Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.text2 {
+  width: 117px;
+  height: 24px;
+  font-family: Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.main-section {
+  background-image: url("images/header-shapes\ mobile.png"), url("images/header-background-mobile.svg");
```

Comment:
Kindly fix the path to the background image. It currently does not show. It's important for the appearance of a page.

This is how your page appears.
<img width="381" alt="Screenshot 2023-01-19 at 15 35 17" src="https://user-images.githubusercontent.com/48546992/213447186-4519d464-03c6-4d61-8871-81afcb39ef0d.png">
This is how the template is:
<img width="381" alt="Screenshot 2023-01-19 at 09 40 33" src="https://user-images.githubusercontent.com/48546992/213447242-72003927-1152-43fd-bb90-b39b6df38cc7.png">